whipper/common/program.py

```diff
@@ -35,6 +37,8 @@
 
 import logging
 logger = logging.getLogger(__name__)
+musicbrainzngs.set_useragent("whipper", whipper.__version__,
```

Comment:
I'm not sure but this statement may be unnecessary because of the `mbngs` import: in that file the useragent is set in the first instructions ([here](https://github.com/whipper-team/whipper/blob/d665fe44c47d7e45a5bd0ebe282071f3475d0173/whipper/common/mbngs.py#L32)).

Reply:
Ahh yeah you're right, i didn't notice that.